infogami/infobase/_json.py

```diff
@@ -23,7 +24,7 @@ def unicodify(d):
     elif isinstance(d, list):
         return [unicodify(x) for x in d]
     elif isinstance(d, str):
-        return d.decode('utf-8')
+        return six.ensure_str(d)
```

Comment:
We're not sure about this one

Reply:
See: https://bugs.launchpad.net/infogami/+bug/231831
unicodify was a stopgap for python...... 2.4 I think we're safe blowing away unicodify